source/_integrations/melnor.markdown

```diff
@@ -0,0 +1,33 @@
+---
+title: Melnor Bluetooth
+description: Instructions on setting up Melnor Bluetooth devices within Home Assistant.
+ha_category:
+  - Switch
+ha_iot_class: Local Polling
+ha_release: 2022.5
+ha_config_flow: true
+ha_codeowners:
+  - '@vanstinator'
+ha_domain: melnor
+ha_platforms:
+  - switch
+ha_integration_type: integration
+---
```

Comment:
Usually we want to tell them they need to turn on the bluetooth integration first

Example `The Govee BLE integration will automatically discover devices once the [Bluetooth](/integrations/bluetooth) integration is enabled and functional.`

Reply:
On it 👍🏻

Reply:
I looked at the Govee docs and that blurb doesn't really jump out. How do you feel about wrapping this in a note block at or near the top of the page?

Reply:
Pushed it out, let's see what it looks like when the renderer is done.

Reply:
I'm not sure we need to call it out since we don't in the other ones and people have been able to figure it out.  I expect it will become less valuable over time as people learn the system

Reply:
Sounds good. I'll match what we're doing elsewhere.